scripts/install-worker.sh

```diff
@@ -193,26 +196,26 @@ EOF
 sudo yum install -y device-mapper-persistent-data lvm2
 
 if [[ ! -v "INSTALL_DOCKER" ]]; then
-  INSTALL_DOCKER=$(vercmp "$KUBERNETES_VERSION" lt "1.25.0" || true)
+  INSTALL_DOCKER=$(vercmp "${KUBERNETES_VERSION}" lt "1.25.0" || true)
 else
   echo "WARNING: using override INSTALL_DOCKER=${INSTALL_DOCKER}. This option is deprecated and will be removed in a future release."
 fi
 
-if [[ "$INSTALL_DOCKER" == "true" ]]; then
+if [[ "${INSTALL_DOCKER}" == "true" ]]; then
   sudo amazon-linux-extras enable docker
   sudo groupadd -og 1950 docker
-  sudo useradd --gid $(getent group docker | cut -d: -f3) docker
+  sudo useradd --gid "$(getent group docker | cut -d: -f3)" docker
 
   # install docker and lock version
-  sudo yum install -y docker-${DOCKER_VERSION}*
+  sudo yum install -y "docker-${DOCKER_VERSION}"*
```

Comment:
Do we really need `*` here?

```bash
# yum search docker
Loaded plugins: priorities, update-motd, versionlock
============================================================================================================ N/S matched: docker =============================================================================================================
pcp-pmda-docker.x86_64 : Performance Co-Pilot (PCP) metrics from the Docker daemon
amazon-ecr-credential-helper.x86_64 : Amazon ECR Docker Credential Helper
docker.x86_64 : Automates deployment of containerized applications
nerdctl.x86_64 : nerdctl is a Docker-compatible CLI for containerd.
oci-add-hooks.x86_64 : Injects OCI hooks as a Docker runtime

  Name and summary matches only, use "search all" for everything.
```

Reply:
The asterisk would allow more flexible values of the `docker_version` template variable to be used, but IMO the user should specify that, we shouldn't hard code it here.

That being said, I consider all Docker code paths to be frozen at this point. This will be removed when 1.23 is EOL and shouldn't be modified until then.

Reply:
I prefer to leave it as it, not going change it's logic in this PR. Just let `shellcheck` pass 😄